configs/material-ui.json

```diff
@@ -47,7 +46,5 @@
     }
   },
   "min_indexed_level": 1,
-  "js_render": true,
```

Comment:
Our docs pages are now server side rendered. This change should speed up the crawling.